project1/project1/import.py

```diff
@@ -0,0 +1,42 @@
+import os
+import datetime
+import csv
+
+from flask import *
+from flask_session import Session
+from sqlalchemy import create_engine
+from sqlalchemy.orm import scoped_session, sessionmaker
+from flask_sqlalchemy import SQLAlchemy
+
+app=Flask(__name__)
+app.config["SQLALCHEMY_DATABASE_URI"] = os.getenv("DATABASE_URL")
+app.config["SQLALCHEMY_TRACK_MODIFICATIONS"] = False
+
+db=SQLAlchemy(app)
+
+class Book(db.Model):
+    _tablename_="books"
+    isbn=db.Column(db.String,primary_key=True)
+    title=db.Column(db.String,nullable=False)
+    author=db.Column(db.String,nullable=False)
+    year=db.Column(db.String,nullable=False)
+
+    def _init_(self,isbn,title,author,year):
+        self.isbn=isbn
+        self.title=title
+        self.author=author
+        self.year=year
```

Comment:
Keep the definations of your DB classes, (Like Books, Users) in separate file and use them wherever needed. 

This is will improve the flexibilty of coding and reduce the redundancy.